examples/index.html

```diff
@@ -1,43 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Examples | CARTO</title>
-  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
-  <link type="text/css" rel="stylesheet" href="../docs/public/styles/jsdoc-default.css">
```

Comment:
My style :cry:

Reply:
it was not being used! I got a 404

Reply:
![cartogl-web](https://user-images.githubusercontent.com/2227572/37355426-10d226d4-26e4-11e8-8e24-9fac21b4e81e.gif)